Gemfile

```
@@ -28,15 +30,16 @@ gem "rspec"
 
 group :development, :test do
   gem "byebug", "~> 11.0", platform: :mri
-  gem "rubocop-faker"
-
   gem "decidim-dev", DECIDIM_VERSION
+  gem "parser", "3.3.0.4"
+  gem "rubocop-ast", "1.31.2"
+  gem "rubocop-faker"
 end
 
 group :development do
   gem "letter_opener_web", "~> 1.3"
   gem "listen", "~> 3.1"
-  gem "spring", "~> 2.0"
+  gem "spring", "4.0"
```

Forcing a version for this gem could give us problems in the future, remove the version here.

```suggestion
  gem "spring"
```